components/article/client/view.coffee

```diff
@@ -184,23 +187,39 @@ module.exports = class ArticleView extends Backbone.View
       imagesLoaded $(this), =>
         $(this).width $(this).children('img').width()
 
-  fillwidth: (el, cb) ->
-    if @$(el).length < 1 or $(window).width() < 700
+  fillwidth: (el, cb=->) ->
+    if @$(el).length < 1 or $(window).width() < 400
       @$(el).parent().removeClass('is-loading')
       return cb()
     $list = @$(el)
+    if $(window).height() > 700
+      newHeight = $(window).height() * .8
     $list.fillwidthLite
       gutterSize: 30
+      targetHeight: newHeight || 600
       apply: (img) ->
         img.$el.closest('li').width(img.width)
       done: (imgs) ->
         # Make sure the captions line up in case rounding off skewed things
         tallest = _.max _.map imgs, (img) -> img.height
         $list.find('.artwork-item-image-container').each -> $(this).height tallest
+        # Account for screens that are both wide & short
+        listWidth = _.map imgs, (img) -> img.width
+        parentWidth = _.reduce(listWidth, (a, b) ->
+                return a + b
+              , 0) + (($($list).children().length - 1) * 30)
+        if $($list).width() > parentWidth
+          $($list).css({'display':'flex', 'justify-content':'center'})
         # Remove loading state
         $list.parent().removeClass('is-loading')
         cb()
 
+  refreshWindowSize: =>
+    $("[data-layout='overflow_fillwidth'] ul").each (i, imgs) =>
+      if $(imgs).children().length > 1
+        @parentWidth = $(imgs).width()
+        @fillwidth imgs
```

Comment:
This is veerry minor but it's also possible to call this as `@fillwidth imgs, ->` which lets you remove the default value. I don't have a preference, so it's just a note.